CP-CPS.md

```diff
@@ -1296,7 +1296,7 @@ No stipulation.
 
 ## 9.7 Disclaimers of warranties
 
-ISRG certificates and services are provided "as-is." ISRG disclaims any and all warranties of any type, whether express or implied, including and without limitation any implied warranty of title, non-infringement, merchantability, or fitness for a particular purpose, in connection with any ISRG service or ISRG certificate.
+Subscribers acknowledge that use of ISRG’s Services is voluntary and at their own risk. ISRG provides Services “as is” and “as available” without warranties of any kind, except as expressly required by the Baseline Requirements.
```

Comment:
Replacing smart quotes with straight quotes:
```suggestion
Subscribers acknowledge that use of ISRG's Services is voluntary and at their own risk. ISRG provides Services "as is" and "as available" without warranties of any kind, except as expressly required by the Baseline Requirements.
```

I'm not sure it's appropriate to say "Subscribers acknowledge" in our CP/CPS -- a document which describes what _we_ do. Maybe we could say that "Our Subscriber Agreement requires that Subscribers acknowledge...". Or we could simply say "The use of ISRG's Services is voluntary and at the Subscriber' own risk."